deployment/ansible/roles/cac-tripplanner.otp-data/tasks/main.yml

```diff
@@ -5,6 +5,16 @@
 - name: Copy Feed Validator
   copy: src=validate_feed.py dest="{{ otp_data_dir }}"
 
+- name: Install AuthBind (for serving OTP on port 80)
```

Comment:
Yeah, @flibbertigibbet told me about it. Seemed like the quickest way to get it running on port 80, though it may be worth switching to nginx at some point if our setup gets more complicated.